styles/Main.module.css

```diff
@@ -23,9 +23,9 @@
 }
 
 .colorB {
-    @apply text-[#1D4ED8];
+    @apply text-[#1d4ed8];
```

Comment:
I think this name for the color is not good, maybe think of a better name like "dark-gray", "light-purple" or "spark-yellow"

Reply:
revisar nuevo color @Israel-Laguan

Reply:
I understood to change the color palette

Reply:
Sorry for the misunderstanding